onnx/backend/test/cmd_tools.py

```diff
@@ -46,12 +46,16 @@ def prepare_dir(path):  # type: (Text) -> None
                     output_dir, 'test_data_set_{}'.format(i))
                 prepare_dir(data_set_dir)
                 for j, input_np in enumerate(inputs):
+                    if input_np.dtype == 'object':
+                        raise TypeError('Sequences not yet supported.')
```

Comment:
Seems we need to extend numpy_helper to add conversion from `list[np.array]` to `SequenceProto` and vice versa. And by using `SequenceProto::SerializeToString()` we should be able to store the values to file.

Reply:
Working on this currently. Should have a PR update out soon.

Reply:
Updated! See numpy_helper for the new functions. Also added support for Maps.